Nikita_Konkov_Application/CSharpAdvanced/CSharpAdvanced/Flowers/CustomFlower.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpAdvanced
+{
+    public class CustomFlower<T> where T : Flower
+    {
+        public int costUp = 15;
+        
+        public string ChangeColor(T flower) 
+        {
+            Console.Write($"Enter new color for {flower.Name}: ");
```

Comment:
Hmm, imagine the situation when you have white flowers and then customer came and said - I want to repaint flower in white. And you will repaint and you will call method change cost. 
Imagine another situation when customer repainted white flowers to blue, but you forgot to call ChangeCost method. You company will loose money